### PR TITLE
home: Show Install button on the homepage

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
FlowForge Cloud is the best and easiest way to validate if FlowForge
works for the use-case envisioned. However, if you're looking to roll
out or use the community edition to evaluate the enterprise edition it's
hard to understand how to start.

### CODE REVIEW

**Comment by ZJvandeWeg** [comment]
Draft as I don't understand how to align buttons at this time.

File: src/index.njk [hunk 1]
```diff
@@ -12,10 +12,10 @@ layout: default
             environment, built around the open-source Node-RED project
         </h3>
         <div class="flex flex-col">
-                <div class="max-w-md m-auto mt-6">
-                    <a class="ff-btn ff-btn--secondary" href="https://app.flowforge.com/account/create">Sign up</a>
-                </div>
-            </form>
+          <div class="max-w-md m-auto mt-6">
```
Comment (knolleary) [inline src/index.njk:15]:
```suggestion
          <div class="max-w-md m-auto mt-6 flex flex-row justify-center space-x-4">
```

Comment (knolleary) [reply src/index.njk:15]: image

File: src/index.njk [hunk 2]
```diff
@@ -26,7 +26,7 @@ layout: default
 <div class="w-full ff-bg-light px-6 pt-12 pb-48 md:pt-32 md:px-0">
     <div class="container m-auto text-center max-w-4xl">
         <div class="m-auto mt-24 w-full text-center">
-            <h2>3,000+ Integrations Available</h2>
+            <h2>3,900+ Integrations Available</h2>
```
Comment (ZJvandeWeg) [inline src/index.njk:29]: Over 4k!

```suggestion
            <h2>4,000+ Integrations Available</h2>
```

**Comment by knolleary** [comment]
Any reason not to mark this ready to review and get merged now?

**Pull request merged**